packages/gateway/test/api.spec.ts

```diff
@@ -73,7 +73,7 @@ describe('Gateway API', () => {
     const domains = new Map()
     domains.set(node, domain)
     repo.setDomains(domains)
-    validator = new OwnershipValidator(signatureRecover, [repo])
+    validator = new OwnershipValidator(1, signatureRecover, [repo])
```

Comment:
What is this 1? Always prefer naming an input through a variable, for code readability.

Reply:
it is the chainId, gonna move it to a variable